components/StepsWizard/index.tsx

```diff
@@ -1,51 +1,44 @@
 import { useState } from 'react';
+import Router from 'next/router';
 import { useBeforeunload } from 'react-beforeunload';
+import { stepKeys } from '../Steps';
 import Link from 'next/link';
-import Router, { useRouter } from 'next/router';
-import * as HttpStatus from 'http-status-codes';
 
-import { steps, stepPath, stepKeys } from '../../components/Steps';
-import { isExpired } from '../../utils/date';
-
-const getAdjacentSteps = (step) => {
-  const currentStep = stepKeys.findIndex((s) => s === step);
-  return {
-    previousStep: currentStep > 0 ? `/step/${stepKeys[currentStep - 1]}` : null,
-    nextStep:
-      currentStep < stepKeys.length
-        ? `/step/${stepKeys[currentStep + 1]}`
-        : null,
-  };
-};
+const StepsWizard = ({ grant, grantSlug, stepId }) => {
+  const [formData, setFormData] = useState<any>({ grantType: grantSlug });
```

Comment:
Can we be more specific with the type here, rather than `any`?

Reply:
Will review as multiple grant types are implemented as sections could potentially be fluid, will need to review how best to implement when we have multiple.